thermal.c

```diff
@@ -93,19 +93,19 @@ static gboolean prepare_netlink(void)
 
 	sock = nl_socket_alloc();
 	if (!sock) {
-		log(TO_ALL, LOG_ERR, "thermal: socket allocation failed.\n");
+		log(TO_ALL, LOG_WARNING, "thermal: socket allocation failed.\n");
```

Comment:
I think these are valid errors, as they should not fail unless extreme circumstances